packages/devtools_app/test/memory/tracing/tracing_view_test.dart

```diff
@@ -88,50 +50,66 @@ void main() {
   const windowSize = Size(2225.0, 1000.0);
 
   group('Allocation Tracing', () {
+    late MemoryDefaultScene scene;
+
     late final CpuSamples allocationTracingProfile;
 
-    setUpAll(() {
+    Future<void> pumpScene(WidgetTester tester) async {
+      await tester.pumpWidget(scene.build());
+      // Delay to ensure the memory profiler has collected data.
+      await tester.pumpAndSettle(const Duration(seconds: 1));
+      expect(find.byType(MemoryBody), findsOneWidget);
+      await tester.tap(
+        find.byKey(MemoryScreenKeys.dartHeapAllocationTracingTab),
+      );
+      await tester.pumpAndSettle();
+    }
+
+    setUpAll(() async {
       final rawProfile = File(
         'test/test_infra/test_data/memory/allocation_tracing/allocation_trace.json',
       ).readAsStringSync();
       allocationTracingProfile = CpuSamples.parse(jsonDecode(rawProfile))!;
     });
 
-    setUp(() {
-      setGlobal(DevToolsExtensionPoints, ExternalDevToolsExtensionPoints());
-      setGlobal(NotificationService, NotificationService());
-      setGlobal(OfflineModeController, OfflineModeController());
-      setGlobal(IdeTheme, IdeTheme());
-      setGlobal(PreferencesController, PreferencesController());
+    setUp(() async {
+      // Modify the character width that will be used to calculate column sizes
+      // in the profiler table. The flutter_tester device uses a redacted font.
+      setAssumedMonospaceCharacterWidth(14.0);
```

Comment:
Can we make a test util for this that we can call without specifying the actual character width each time? This should be the same for all the tests, right?